student/views/slack.py

```diff
@@ -0,0 +1,43 @@
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+import json
+import threading
+
+from authentication.models import StudentUser
+from student.library.slack import Slack
+
+
+@csrf_exempt
+def event_hook(request):
+    request_body_json_dict = json.loads(request.body.decode('utf-8'))
+
+    if 'type' in request_body_json_dict and request_body_json_dict['type'] == 'url_verification':
+        return JsonResponse({'challenge': request_body_json_dict['challenge']}, safe=False)
+
+    if 'event' in request_body_json_dict:
+        event = request_body_json_dict['event']
+
+        if event['type'] == 'team_join':
+            thread = threading.Thread(target=team_join_event, args=(event,))
+            thread.start()
+
+            return HttpResponse(status=200)
+
+def team_join_event(event):
```

Comment:
Should we add comment above this to clarify what this function does without having to read code?